WalletWasabi.Fluent/ViewModels/NavBar/NavBarViewModel.cs

```diff
@@ -206,8 +204,10 @@ private void SetSelectedItem(NavBarItemViewModel? value)
 			}
 		}
 
-		private void CurrentPageChanged(NavBarItemViewModel x, WalletManagerViewModel walletManager)
+		private void CurrentPageChanged(NavBarItemViewModel x)
 		{
+			var walletManager = UiServices.WalletManager;
```

Comment:
Use UiServices.WalletManager instead of local variable

Reply:
done.